app/controllers/countries_controller.rb

```diff
@@ -3,7 +3,7 @@ def index
   end
 
   def show
-    @country = Country.find(params[:id])
+    @country = GeoEntity.find_by(iso3: params[:id])
```

Comment:
@Levia I already tried to get iso3 codes being used instead of the id however, we don't know yet what the url should look like, so I think I will revert this change!

Reply:
Actually, since I changed to using `GeoEntity` to contain both `countries` and `regions` the existing code to link to a particular `id` won't be the same and it will not link to the correct country.

Reply:
I just fixed that now